src/systems/ec/implementations/react/packages/gallery/src/GalleryItem.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Icon from '@ecl/ec-react-component-icon/Icon';
+
+const GalleryItem = ({ item, className, ...props }) => (
+  <li {...props} className={classnames(className, 'ecl-gallery__item')}>
+    <a
+      href={item.src}
+      className="ecl-gallery__item-link"
+      aria-label={item.alt}
+      data-ecl-gallery-item
+    >
+      <figure className="ecl-gallery__image-container">
+        <img src={item.src} alt={item.alt} className="ecl-gallery__image" />
+        {!!(item.icon && item.icon.shape) && (
+          <Icon
+            {...item.icon}
+            size="l"
+            className={classnames(
+              item.icon.className,
+              'ecl-gallery__image-icon'
+            )}
+          />
+        )}
+
+        <figcaption
+          className="ecl-gallery__description"
+          data-ecl-gallery-description
+        >
+          {item.description}
+          {!!(item.icon && item.icon.shape) && (
+            <Icon
+              {...item.icon}
+              size="l"
+              className={classnames(
+                item.icon.className,
+                'ecl-gallery__description-icon'
+              )}
+            />
+          )}
+        </figcaption>
+
+        <div className="ecl-gallery__meta" data-ecl-gallery-meta hidden>
```

Comment:
I would suggest to get rid of this div and rely on data-attributes
https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes